classes/user/Collector.php

```diff
@@ -425,6 +438,15 @@ public function getQueryBuilder(): Builder
 
         $query = DB::table('users', 'u')
             ->select('u.*')
+            // Adds active submissions that are assigned to each user
+            ->when($this->includeActiveSubmissions != null, fn (Builder $query) => $query->selectSub(function ($query) {
+                $query->from('stage_assignments as sa')
+                    ->join('submissions as s', 'sa.submission_id', '=', 's.submission_id')
+                    ->whereColumn('sa.user_id', 'u.user_id')
+                    ->where('sa.user_group_id', '=', $this->includeActiveSubmissions['user_group_id'])
+                    ->whereNotIn('s.status', [PKPSubmission::STATUS_PUBLISHED, PKPSubmission::STATUS_DECLINED])
```

Comment:
I think we should also exclude incomplete submissions

Reply:
The incomplete submissions have been excluded as well.